### PR TITLE
Added header

### PR DESCRIPTION
**Pull request by younessoudouh**
-Add App header with title and icons

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,10 +1,13 @@
 import React from "react";
 import ReactDOM from "react-dom/client";
 import App from "./App";
+import { BrowserRouter as Router } from "react-router-dom";
```
[the remark this answers is not in the document]
Comment (younessoudouh) [reply src/index.js:4]: to change  BrowserRouter name to Router,  I think Router is easy to read an to write than BrowserRouter.

File: src/Componnets/header/Header.css [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+.title {
+    text-transform: uppercase;
+    font-family: "Montserrat", sans-serif;
+    font-size: 5vw;
+    color: #fff;
+    font-weight: 100;
+}
+
+.Clapperboard {
+    width: 70px;
```
[the remark this answers is not in the document]
Comment (younessoudouh) [reply src/Componnets/header/Header.css:23]: img!

File: src/App.jsx [hunk 3]
```diff
@@ -1,9 +1,15 @@
 import "./App.css";
+import { Header } from "./Componnets/header/Header";
+import { Trending } from "./pages/trending/Trending";
+import { Routes, Route } from "react-router-dom";
 
 function App() {
```
Comment (faouzioudouh) [inline src/App.jsx:6]: Named export please

File: src/Componnets/header/Header.css [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    background-color: #39445a;
+    padding: 11px 0;
+    box-shadow: 0 1px 5px #000;
+    z-index: 100;
+    align-items: center;
+}
+
+.title {
+    text-transform: uppercase;
+    font-family: "Montserrat", sans-serif;
```
Comment (faouzioudouh) [inline src/Componnets/header/Header.css:16]: Font-family should not be added to each element.

File: src/Componnets/header/Header.css [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+    font-weight: 100;
+}
+
+.clapperboard {
+    width: 70px;
+    height: 70px;
+    margin-right: 20px;
+}
+
+.filmprojector {
```
Comment (faouzioudouh) [inline src/Componnets/header/Header.css:28]:
```suggestion
.film-projector {
```

File: src/Componnets/header/Header.css [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+
+.title {
+    text-transform: uppercase;
+    font-family: "Montserrat", sans-serif;
+    font-size: 72px;
+    color: #fff;
+    font-weight: 100;
+}
+
+.clapperboard {
```
Comment (faouzioudouh) [inline src/Componnets/header/Header.css:22]:
```suggestion
.clapper-board {
```

File: src/Componnets/header/Header.css (hunk 4 above)
Comment (younessoudouh) [reply src/Componnets/header/Header.css:16]: yes

File: src/pages/trending/Trending.jsx [hunk 7]
```diff
@@ -4,7 +4,7 @@ import "./Trending.css";
 export const Trending = () => {
   return (
     <div className="trending">
-      <h1>Trending Today</h1>
+      <h1 className="trending-title">Trending Today</h1>
```
Comment (faouzioudouh) [inline src/pages/trending/Trending.jsx:7]: Since the same styles will be used on all pages, can you give it a generic classname?
Like: `page-title` ?

**Review by faouzioudouh — APPROVED** [review]

File: src/pages/trending/Trending.jsx (hunk 7 above)
Comment (younessoudouh) [reply src/pages/trending/Trending.jsx:7]: yes

**Pull request merged**